### PR TITLE
Follow-up the PR #3503(Show all endstops on LCD in checkHitEndstops), etc

### PR DESCRIPTION
**Pull request by ghost**
Follow-up the PR #3503(Show all endstops on LCD in checkHitEndstops), etc

・Put all of the translatable strings (except strings that it makes no sense to translate) into all of the language files
・Remove strings that [it makes no sense to translate](https://github.com/MarlinFirmware/Marlin/pull/3550#discussion_r60059265) (MSG_E1, MSG_E2, MSG_E3, MSG_E4)
  note: This removal erase the customized strings of MSG_E1 ~ E4 in language_gl.h.
・Remove abolished strings (MSG_NOZZLE1, MSG_NOZZLE2)
・Remove duplicated strings
・Arrange the strings in unified order
・Adjust spacing

About Japanese language files:
・Add readable translated strings in comment
・Add Japanese translation.

```
MSG_LCD_ENDSTOPS "エンドストップ"
```

(utf8 version only. Because non-utf8 version ("ｴﾝﾄﾞｽﾄｯﾌﾟ") over 8 characters)

### CODE REVIEW

File: Marlin/language_an.h [hunk 1]
```diff
@@ -59,21 +60,35 @@
...
 #define MSG_SWITCH_PS_OFF                   "Desenchegar Fuent"
 #define MSG_EXTRUDE                         "Extruir"
 #define MSG_RETRACT                         "Retraer"
 #define MSG_MOVE_AXIS                       "Mover Eixes"
+#define MSG_LEVEL_BED                       "Level bed"
 #define MSG_MOVE_X                          "Move X"
 #define MSG_MOVE_Y                          "Move Y"
 #define MSG_MOVE_Z                          "Move Z"
 #define MSG_MOVE_E                          "Extruder"
+#define MSG_MOVE_E1                         "1"
```
Comment (jbrazio) [inline Marlin/language_an.h]: The following directives were left out on propose, makes no sense to translate them:
`MSG_H1 MSG_H2 MSG_H3 MSG_H4 MSG_MOVE_E1 MSG_MOVE_E2 MSG_MOVE_E3 MSG_MOVE_E4 MSG_N1 MSG_N2 MSG_N3 MSG_N4 MSG_DIAM_E1 MSG_DIAM_E2 MSG_DIAM_E3 MSG_DIAM_E4 MSG_E1 MSG_E2 MSG_E3 MSG_E4`

File: Marlin/language_an.h [hunk 2]
```diff
@@ -148,8 +172,19 @@
 #define MSG_BABYSTEP_Y                      "Babystep Y"
 #define MSG_BABYSTEP_Z                      "Babystep Z"
 #define MSG_ENDSTOP_ABORT                   "Endstop abort"
+#define MSG_HEATING_FAILED_LCD              "Heating failed"
```
Comment (jbrazio) [inline Marlin/language_an.h:155]: When a language is missing a translation for one directive it's better **not to include** the English directive on the language file as we now have a script which searches for [missing translations](https://github.com/MarlinFirmware/Marlin/blob/RCBugFix/Marlin/scripts/findMissingTranslations.sh) and outputs them.

The script is unable to detect when a language file has the **directive defined in English** thus makes the language files hard to maintain.

File: Marlin/language_an.h (hunk 1 above)
Comment (ghost) [inline Marlin/language_an.h]: I've updated this PR just now. It removes those directives from all of the language files.

Comment (jbrazio) [inline Marlin/language_an.h]: Thank you.

**Comment by ghost** [comment]
This PR is temporary closed until rework to be completed.

File: Marlin/language_an.h (hunk 2 above)
Comment (ghost) [inline Marlin/language_an.h:155]: I've coped it by re-opened new PR #3559.

**Comment by ghost** [comment]
I opened new PR #3559 as retaken version of this PR.
Because I can't reopen this PR after `rebase -i` and `push -f`...

**Comment by thinkyhead** [comment]
You don't need to re-open a PR if you do `git rebase -i RCBugFix` and `git push -f`. Just don't close it in the first place. The PR is updated as soon as you do `git push -f`.

**Pull request closed without merging**